spec/struct.dd

```diff
@@ -831,6 +831,75 @@ void main()
         qualified with the more restrictive qualifier, which is `immutable`.
         )
 
+        $(P The postblits `__fieldPostblit` and `__aggrPostblit`
+        are generated without any implicit qualifiers and are not considered
+        struct members. This leads to the situation where qualifying an
+        entire struct declaration does not have any impact on the above
+        mentioned postblits. However, since `__xpostblit` is a member of
+        the struct and an alias of one of the other postblits, the qualifiers
+        applied to the struct will affect the aliased postblit:)
+
+---
```

Comment:
`$(SPEC_RUNNABLE_EXAMPLE_FAIL`

Reply:
@RazvanN7 can you please add this when you get the chance?